dist/bump/index.js

```diff
@@ -12333,12 +12333,21 @@ exports.getTags = getTags;
  */
 function getAssociatedPullRequests(sha) {
     return __awaiter(this, void 0, void 0, function* () {
-        const { data: prs } = yield getOctokit().rest.repos.listPullRequestsAssociatedWithCommit({
-            owner: OWNER,
-            repo: REPO,
-            commit_sha: sha,
-        });
-        return prs;
+        try {
+            const { data: prs } = yield getOctokit().rest.repos.listPullRequestsAssociatedWithCommit({
+                owner: OWNER,
+                repo: REPO,
+                commit_sha: sha,
+            });
+            return prs;
+        }
+        catch (error) {
+            if (error.message !== "Resource not accessible by integration") {
```

Comment:
Not sure if they (Github) made a specific error class for this, if so, I would rather depend on the error class
```
if(error instanceof GHError) {
```
Then this error prone string comparison

Reply:
Hmm, tried to google it myself, but didn't find it.

Reply:
Unfortunately it isn't :(